experimenter/experimenter/targeting/constants.py

```diff
@@ -1156,6 +1156,20 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+NOT_IMPORT_INFREQUENT_ROLLOUT = NimbusTargetingConfig(
+    name="Not in Import Infrequent Rollouts",
+    slug="not_import_infrequent_rollout",
+    description="Exclude users in the import infrequent rollouts",
+    targeting="""(activeRollouts intersect [
+            'import-infrequent-rollout-make-yourself-at-home',
+            'updated-import-infrequent-rollout-make-yourself-at-home-copy'
+        ])|length == 0""",
```

Comment:
Eh… the resulting json has `\n` which is fine but prevents easy copy/paste into jexl evaluators. Switching to plain strings instead of multiline.

Better?
![intersect jexl](https://user-images.githubusercontent.com/438537/233745822-ee799795-a178-4981-b9ae-e3d581471a76.png)